docs/reference/hhfab-cli.md

```diff
@@ -0,0 +1,152 @@
+
+# Fabricator CLI Reference
```

Comment:
Why do we need it if it's available by running `hhfab --help` or same on subcommand?  My main concern that it'll get outdated immediately.